tests/nodes/tests_Node.cpp

```diff
@@ -200,3 +200,13 @@ TEST_CASE("IRI UUID") {
     CHECK(uuid != uuid2);  // note: non-deterministic but should basically never fail
     CHECK(regex::Regex{"^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$"}.regex_match(uuid.identifier()));
 }
+
+TEST_CASE_TEMPLATE("IRI/BlankNode::find", T, IRI, BlankNode) {
```

Comment:
This test case will fail if we ever implement bnode validation (I think). Please seperate the 2 cases

Reply:
I assume separating the data of the 2 is enough.
If not, we might want to wait until we know what bnode validation looks like.